<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QWidget{background-color: rgb(25,25, 40);}

#cancelBtn:hover {
background-color:red;
}
#doneBtn:hover{
background-color:rgb(105, 95, 20);
}

#confirmBtn:hover{
background-color:rgb(105, 95, 20);
}

QLabel{
color: rgb(231, 231, 231);
      font: 12pt &quot;Verdana&quot;;
}</string>
       </property>
       <widget class="QLabel" name="sNameLbl">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>210</y>
          <width>161</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Mode of Payment:</string>
        </property>
       </widget>
       <widget class="QLabel" name="sContactNoLbl">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>210</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;CASH&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>370</y>
          <width>141</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(231, 231, 231);
font: 17pt &quot;Verdana&quot;;
border: 2px solid orange;
padding: 5px;
border-radius: 3px;
opacity: 200;
</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="confirmBtn">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>370</y>
          <width>141</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(231, 231, 231);
font: 17pt &quot;Verdana&quot;;
border: 2px solid orange;
padding: 5px;
border-radius: 3px;
opacity: 200;
</string>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
       <widget class="QLabel" name="destinationLbl">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>520</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Reference No.:</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QLabel" name="feeLbl">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>300</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Fee:</string>
        </property>
       </widget>
       <widget class="QLabel" name="showFee">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>300</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;125.00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="totalLbl_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Sub Total:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="doneBtn">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>590</y>
          <width>141</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(231, 231, 231);
font: 17pt &quot;Verdana&quot;;
border: 2px solid orange;
padding: 5px;
border-radius: 3px;
opacity: 200;
</string>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
       <widget class="QLabel" name="refNoShow">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>520</y>
          <width>291</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="totalLbl">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>260</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="showTotal">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>260</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="showSubTotal">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>170</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>591</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
padding-left: 140px;
font: 22pt &quot;Courier New&quot;;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#191928;&quot;&gt;J4 Courier Center&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
